src/main/resources/templates/mail/cratead-tasksqty-email.html

```diff
@@ -93,7 +93,10 @@
                                     <td style="font-family: sans-serif; font-size: 14px; vertical-align: top;">
                                         <p th:if="${is_friend}" style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Yo <span th:text="${admin_config.getAdminName()}"></span>,</p>
                                         <p th:unless="${is_friend}" style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Dear <span th:text="${admin_config.getAdminName()}"></span>,</p>
-                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;" th:text="${message}"></p>
+                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">
+                                            Currently in database you got: <span th:text="${message}"></span>
+                                            <p th:if="${message == 1}"> task.
```

Comment:
Nazwij tę zmienną `tasksCount` albo podobnie. `message` nie jest już adekwatną nazwą.
Dlaczego korzystasz z taga `<p>`? Tag p to paragraf (akapit), jest tagiem blokowym, czyli m. in. wymusza nową linię (chyba że został inaczej ostylowany przez CSS, ale to nienaturalne). Lepiej użyć `<span>`, to jest tag typu inline.